README.md

```diff
@@ -169,219 +169,225 @@ you find it in your resource folder<BR>
-* Pam's Harvestcraft 1.7.10-lb-1.0.6-GTNH<BR>
-* Pam's Harvestcraft Nether 1.7.10a<BR>
-* Project Blue 1.1.6<BR>
-* Project Red-All 4.7.0pre14.18<BR>
-* Questbook 2.1.1<BR>
-* Railcraft 9.12.2.1<BR>
-* Random Things 2.2.4<BR>
-* Realistic World Gen 1.3.2<BR>
-* ReAuth 3.5.0<BR>
-* RemoteIO 2.4.2<BR>
-* RF Expansion 3.0.29(disabled till an update is online)<BR>
-* Rogue like Dungeons 1.5.1<BR>
-* Resource Loader 1.3.0<BR>
-* SG Craft 1.13.9<BR>
-* Sleeping Bag 0.1.1<BR>
-* Special Mobs 3.3.8<BR>
-* Spice of Life 2.0.0-carot-beta-7<BR>
-* Standard Expansion 3.0.185-GTNH<BR>
-* Steve's Carts 2.0.0.20b<BR>
-* Steve's Factory Manager ALPHA97-GTNH<BR>
-* Steve's Factory Manager Addon 0.10.17<BR>
-* Storage Drawers 1.11.7-gtnh<BR>
-* Storage Drawers Wood Addons 1.1.2c<BR>
-* Super Solar Panel 1.1.2<BR>
-* Super Tic 1.2.1<BR>
-* Tainted Magic r7.6<BR>
-* TC Node Tracker 1.1.2<BR>
-* Tec Tech 4.10.6<BR>
-* Thaumcraft 4.2.3.5<BR>
-* Thaumcraft Mob Aspects 2B-snapsh17<BR>
-* Thaumcraft-NEI-Additions 1.1.0.1<BR>
-* ThaumcraftNEIPlugins 1.7a<BR>
-* Thaumic Bases 1.4.25.0<BR>
-* Thaumic Energistics 1.1.3.12<BR>
-* Thaumic Exploration 1.1.76<BR>
-* Thaumic Horizon 1.2.0.4<BR>
-* Thaumic Machina 0.2.1<BR>
-* Thaumic Tinkerer 2.5.558-GTNH<BR>
-* Thaumcraft Tweaks 1.3.2<BR>
-* TiCTooltips 1.2.5<BR>
-* Tinker's Construct 1.9.0.1.Build-11-GTNH<BR>
-* Tinker's Gregworks 1.0.19-GTNH<BR>
-* Tinker's Defense 1.2.1d<BR>
-* TMechworks 0.2.16.2<BR>
-* Too Much Loot 4.0.3.B2<BR>
-* Toro Health 1.0.0<BR>
-* Translocator 1.1.2.16<BR>
-* Travellers Gear 1.16.7<BR>
-* Twillight Forest 2.3.8.7-dev<BR>
-* Ulity Worlds 1.0.9e<BR>
-* Universal Singularities Version 8.6.3<BR>
-* Waila 1.5.10<BR>
-* Waila Harvestability 1.1.7<BR>
-* Waila Plugins 0.2.3.2-17<BR>
-* Wawla 1.0.5.120<BR>
-* Warp Theory 1.1.1-GTNH<BR>
-* Wireless Redstone CBE 1.4.2-GTNH<BR>
-* Witchery 0.24.1<BR>
-* Witchery Extras 1.1.1<BR>
-* Witching Gadgets 1.2.13-GTNH<BR>
-* Ztones 2.2.2custom<BR>
-
-## Libs and Cores<BR>
-* AsieLib 0.4.9<BR>
-* BdLib 1.9.3.109<BR>
-* Brandon's Core 1.0.0.12<BR>
-* CodeChickenCore 1.07.47<BR>
-* CodeChickenLib 1.1.3.140<BR>
-* CofhCore 3.1.4.329<BR>
-* CofhLib 1.2.1.185<BR>
-* Crop Loader Core 0.1.4<BR>
-* Dummy Core 1.13<BR>
-* Ender Core 0.2.0.39<BR>
-* Forge Multipart 1.2.1.6<BR>
-* InpureCore 1.0.0B9-62<BR>
-* Malisis Core 0.14.3<BR>
-* MicdoodleCore 3.0.13-GTNH<BR>
-* Mantle 0.3.2b<BR>
-* MrTJPCore 1.1.0.33<BR>
-* Open Mods Lib 0.10.1<BR>
-* p455w0rdslib 1.0.4<BR>
-* Player api 1.4<BR>
-* Sponge Mixins 1.3.2<BR>
-* Structure Lib 1.0.7<BR>
-* Wanion Lib 1.8<BR>
-* YamCore 0.5.78<BR>
+## Modlist - NewHorizons
+
+| Name | Version |
+| --- | --- |
+| Advanced Solar Panels | 3.5.1a |
+| [Adventure Backpack](https://github.com/GTNewHorizons/AdventureBackpack2) | 0.9h15-GTNH |
+| [AE2 Stuff](https://github.com/GTNewHorizons/ae2stuff/) | 0.5.1.11_GTNH |
+| [AE2 Wireless Crafting Table](https://github.com/GTNewHorizons/WirelessCraftingTerminal) | 1.8.7.8-GTNH |
```

Comment:
Wireless Crafting Terminal - not table ;)

Reply:
Hey, it was already like that! I'll fix it, though.